packages/webapp/src/components/Filter/FilterMultiSelectV2/index.tsx

```diff
@@ -116,7 +116,7 @@ export const FilterMultiSelectV2 = ({
 
   useEffect(() => {
     if (shouldReset) {
-      setValue(options.filter((option) => option.default));
+      setValue([]);
```

Comment:
The default filters in `options` are the filters that were initially active when the filter modal is opened, empty array removes all selected options